### DISCUSSION TITLE
Pack multiple font styles into one file?

### DISCUSSION THREAD

**Discussion by ShikyC** [fonttools/fonttools#3296] [Q&A] [answered]
I created the regular, italic, bold and bold-italic styles for my font using FontBuilder and saved them as individual OTF files. Can I pack them together into one file that contains all the styles? For some reasons I can't use the FontForge Python package so I have to use FontTools.

**Comment by behdad** [comment, marked as answer]
Check out `TTCollection`. Or from the commanline:
```
fonttools ttLib --help
```

**Reply by ShikyC** [reply]
Thanks! `TTCollection` is exactly what I need. Very neat interface to pack styles together.

```
collection = TTCollection()
for f in fonts:
    collection.fonts.append(f)
collection.save("all-in-one.ttf")
```

**Reply by canlzz** [reply]
it not all, i found [f1.ttf, f2.ttf] in fonts, collection.save("all-in-one.ttf"), but only f2 in all-in-one.ttf
